virtual/integration/main_test.go

```diff
@@ -128,7 +128,7 @@ func NewServer(
 	receiveCallback func(meta payload.Meta, pl payload.Payload) []payload.Payload,
 	mManager machinesmanager.MachinesManager) (*Server, error) {
 
-	traceID := "main_" + utils.RandTraceID()
+	traceID := utils.RandTraceID() + "_main"
```

Comment:
What for?

Reply:
to be able to track this trace with jaeger (I cut everything after 32 symbols). But still have them verbose in logs